core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1868,144 +1868,53 @@ paths:
-      description: |
-        Create method configurations.
-
-        #### Expanded discussion on the methodRepoMethod field
-
-        To create a method config for an Agora method, `"sourceRepo": "agora"` is optional; the following is sufficient:
-        ```
-        "methodRepoMethod": {
-          "methodNamespace": "namespace",
-          "methodName": "name",
-          "methodVersion": 1
-        }
-        ```
-        To create a method config for a Dockstore method, `"sourceRepo": "dockstore"` is required:
-        ```
-        "methodRepoMethod": {
-          "sourceRepo": "dockstore",
-          "methodPath": "path",
-          "methodVersion": "version"
-        }
-        ```
-        You may also use the URI on its own to create a config referencing any supported repo (currently Agora and Dockstore):
-        ```
-        "methodRepoMethod": {
-          "methodUri": "agora://namespace/name/1"
-        }
-        ```
-        ```
-        "methodRepoMethod": {
-          "methodUri": "dockstore://path/version"
-        }
-        ```
-        The system is specified to check for a URI first before falling back to the legacy fields. Unsupported repos will return a 400 Bad Request.
-      operationId: create_method_configuration
+        - entities
+      summary: Change attribute name for an entity type
+      description: Change the attribute name for an entity type in a workspace.
+        The attribute must already exist, and the new name cannot exist. If there
+        is a conflict (between the new name and another name), the update will fail,
+        and no records will be updated.
+      operationId: rename_entity_attributes
       parameters:
         - $ref: '#/components/parameters/workspaceNamespacePathParam'
         - $ref: '#/components/parameters/workspaceNamePathParam'
+        - $ref: '#/components/parameters/entityTypePathParam'
+        - $ref: '#/components/parameters/attributeNamePathParam'
       requestBody:
-        description: Method Configuration contents
+        description: New attribute name
         content:
           'application/json':
             schema:
-              $ref: '#/components/schemas/MethodConfiguration'
+              $ref: '#/components/schemas/NewAttributeNameDefinition'
         required: true
       responses:
-        201:
-          description: Successful Request
-          headers:
-            Location:
-              description: The path to the newly created method configuration
-              schema:
-                type: string
-          content:
-            'application/json':
-              schema:
-                $ref: '#/components/schemas/ValidatedMethodConfiguration'
+        204:
+          description: Attribute name updated
+          content: { }
         403:
-          description: Invalid attribute namespace(s) in outputs
+          description: User does not have permission to update Attributes in the namespace
```

Comment:
nit: remove "in the namespace" here, that will be confusing to most users. How about "User does not have permission to update attribute names"?